### ISSUE TITLE
[CAIA Intake] Unified Appointment Experience: MyHealth - New Community Care Direct Scheduling Tool

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#81267]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Appointments
- **OCTO product owner:** Kay Lawyer
- **Product name:** Community Care Direct Scheduling
- **Product manager:**  Jeff Roof
- **Slack channel:** #appointments-cc-direct-scheduling
- **Dedicated content writer on your team (if you have one):** shared
- **Dedicated a11y specialist on your team (if you have one):** shared
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [x] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

[CC Direct Scheduling initiative brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/initiatives/community-care-direct-scheduling/readme.md#cc-direct-scheduling-initiative-brief)

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [x] Yes
- [ ] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

A pilot (TBD) must be released in December 2024, i.e., a set of Veterans can direct schedule community care providers in the authenticated Appointments experience. And, cancel those appointments. 

[User flow options](https://www.figma.com/file/DsRXEFiYLCFnY5nBkp9Dc4/CC-Referral-%7C-Appointments-FE?type=design&node-id=75%3A1321&mode=design&t=4cLdIKzmoHB5pPep-1)

[CC direct scheduling high level visual data flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1712435187811/f3ab95fd81a0a49af7d20f4629e9fee0356bb8d9?sender=u37bb983bd3fc3cc00c7d3286)

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent (Scheduled for Thursday, 4/25)
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** #80703

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.

[EPS-VAOS Integration Vision](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/initiatives/community-care-direct-scheduling/discovery/eps-vaos-integration-vision.md)

[User flow options](https://www.figma.com/file/DsRXEFiYLCFnY5nBkp9Dc4/CC-Referral-%7C-Appointments-FE?type=design&node-id=75%3A1321&mode=design&t=4cLdIKzmoHB5pPep-1)

[CC direct scheduling high level visual data flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1712435187811/f3ab95fd81a0a49af7d20f4629e9fee0356bb8d9?sender=u37bb983bd3fc3cc00c7d3286)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [x] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [x] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA a11y tasks
- [x] Review Design Intent Feedback from Brian #81745
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/86740
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/86183
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/82200
```

**Comment by benbrasso-agile6** [comment]
FYI @sterkenburgsara @laurwill
